### ISSUE TITLE
Different encoding behavior between browser & node.

### ISSUE THREAD

**Issue by f15gdsy** [nuxt/nuxt.js#946]
Hi,

I used axios to make requests to an API server.
The request url sent in browser & development server is correct, which is only encoded once (by axios, I think).
However, the request url sent in production server is wrong, which is encoded twice.

For example, I have a request url with a query
```
父亲节
```
which is encoded correctly into (in browser & development server by axios)
```
%E7%88%B6%E4%BA%B2%E8%8A%82
```
but in production server, the encoded string will be encoded again into (not sure by whom)
```
%25E7%2588%25B6%25E4%25BA%25B2%25E8%258A%2582
```

I'm using nuxt build and nuxt start command to run the Nuxt render server.

Any idea about what could be wrong?

**Comment by pi0** [comment]
Hi. What do you mean about term `encoded`?

**Comment by f15gdsy** [comment]
@pi0 Hi, I have updated the description. And by ```encoded``` I mean something like ```encodeURI``` in browser.

**Comment by pi0** [comment]
@f15gdsy Just made some experiments.

Code: https://glitch.com/edit/#!/large-part?path=pages/index.vue:10:0
Load this page to make requests : https://large-part.glitch.me/?foo=父亲节
Request log: http://mockbin.org/bin/808ef428-f0fe-4451-b920-dd15342849c3/log

Production: http://mockbin.org/bin/808ef428-f0fe-4451-b920-dd15342849c3/log#entry-20
Dev: http://mockbin.org/bin/808ef428-f0fe-4451-b920-dd15342849c3/log#entry-24

Don't know why Chinese encoding is broken, but both report same URL.

**Comment by pi0** [comment]
Working on a better test soloution

**Comment by f15gdsy** [comment]
For ASCII characters it should be fine, as after encoding, they remains the same. But for UTF-8 characters it will be transformed.

**Comment by pi0** [comment]
I've updated tests suit. Seems calling `encodeURI` when `axios.get()` is enough and echo back API reports 1 time encoding. Please compare implementations.  Is your usage having different behavior?

**Comment by f15gdsy** [comment]
@pi0 Thank you very much for your help! 

I still could not figure it out. 
I have compared the url for the request and the actual url in the request header. The first url in browser, dev server, and production server are the same. But the actual url in the request header for browser, dev server are correct, but for production server it's not.

So for now, I just moved request with Chinese params to browser execution only. This solves the problem through SEO may hurts.
